### ISSUE TITLE
flagged

### ISSUE THREAD

**Issue by ruskiballer** [kekeds/funcaptcha-solver#2]
bro tokens generated with this are flagged... how to solve?

**Comment by zerlokker** [comment]
just unflag it yourself 💀

**Comment by ruskiballer** [comment]
> just unflag it yourself 💀

thanks for the great advise

**Comment by ruskiballer** [comment]
> > > just unflag it yourself 💀
> > 
> > 
> > thanks for the great advise
> 
> welcome. but i said that cuz its just cancer to unpatch...

Absolutely. I just don't understand why the repo owner published this and claimed that it works even though it doesn't. Many people are looking for a working service, so he would have been better advised to fix his code instead of releasing this nonsense.

**Comment by rocketprogrammer** [comment]
> > > > just unflag it yourself 💀
> > > 
> > > 
> > > thanks for the great advise
> > 
> > welcome. but i said that cuz its just cancer to unpatch...
> 
> Absolutely. I just don't understand why the repo owner published this and claimed that it works even though it doesn't. Many people are looking for a working service, so he would have been better advised to fix his code instead of releasing this nonsense.

It did work at the time it was released lmao

**Comment by VendorAttestation** [comment]
It's easy to unpatch get good!

**Comment by VendorAttestation** [comment]
> > It's easy to unpatch get good!
> 
> did you get it work? whats your tg ready to pay

https://github.com/kekeds/funcaptcha-solver/pulls

here's a start if anything else gets leaked ill release updates 👌 but for now will keep rest private

**Comment by Somberor** [comment]
> > > It's easy to unpatch get good!
> > 
> > 
> > did you get it work? whats your tg ready to pay
> 
> #6
> 
> here's a start if anything else gets leaked ill release updates 👌 but for now will keep rest private

hey are you selling the solution you have atm?

**Comment by VendorAttestation** [comment]
> > > > It's easy to unpatch get good!
> > > 
> > > 
> > > did you get it work? whats your tg ready to pay
> > 
> > 
> > #6
> > here's a start if anything else gets leaked ill release updates 👌 but for now will keep rest private
> 
> hey are you selling the solution you have atm?

Wdym Solution

https://github.com/kekeds/funcaptcha-solver/pull/7 more updates guys

**Comment by iqnp** [comment]
> > > > > It's easy to unpatch get good!
> > > > 
> > > > 
> > > > did you get it work? whats your tg ready to pay
> > > 
> > > 
> > > [#6](https://github.com/kekeds/funcaptcha-solver/pull/6)
> > > here's a start if anything else gets leaked ill release updates 👌 but for now will keep rest private
> > 
> > 
> > hey are you selling the solution you have atm?
> 
> Wdym Solution
> 
> [#7](https://github.com/kekeds/funcaptcha-solver/pull/7) more updates guys

more updates if possible?
